doc/whats_new/v0.22.rst

```diff
@@ -555,8 +560,8 @@ Changelog
   Dockès <jeromedockes>`.
 
 - |Feature| Added multiclass support to :func:`metrics.roc_auc_score` with
-  corresponding scorers 'roc_auc_ovr', 'roc_auc_ovo', 'roc_auc_ovr_weighted', 
-  and 'roc_auc_ovo_weighted'. :pr:`12789` and :pr:`15274` by 
+  corresponding scorers `roc_auc_ovr`, `roc_auc_ovo`, `roc_auc_ovr_weighted`, 
+  and `roc_auc_ovo_weighted`. :pr:`12789` and :pr:`15274` by 
```

Comment:
in fact they should string so it should for instance `'roc_auc_ovo_weighted'`. Otherwise LGTM

Reply:
Grrrr I have reintroduced the warnings... :(

Reply:
Ah no, wait! those are the warnings I have fixed in  #15560! :)